.changeset/many-dingos-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mds-core/mds-geography-service": patch
+---
+
+Make getPublishedGeographies bypass RPC by default, adjust GeographyWithMetadataDomainModel to be more accurate
```

Comment:
Bypassing RPC should _not_ be the _default_ behavior; it should be the exception.